### DISCUSSION TITLE
Number of Photons at a certain depth

### DISCUSSION THREAD

**Discussion by ABoamfa** [ankrh/MCmatlab#13] [General]
Hello,

First of all many thanks for the great package! I have a small question I would like to ask.
I am modelling the light propagation through a slab of (homogenous) tissue. I have modelled a source and detector pair that are separated by 20mm. I only keep the photons that arrive at the detector, and I disregard the rest. Now would like to know from these photons, how many/what percentage has reached depth xx (e.g. 10mm), I am interested in the whole x-y plane here. I can define the normalised fluence rate of a plane at each depth. But is there any way to determine the actual number of photons that passed a certain depth?
Thanks in advance for any help!

**Comment by ankrh** [comment]
Yes! You can use the new deposition criteria. See example 25.

You would split your homogenous medium into two media with identical properties, "surface tissue" and "deep tissue". Then you could use something like these deposition criteria:

model.MC.depositionCriteria.minInterfaceTransitions = 1;
model.MC.depositionCriteria.minMediumIdxToConsider = 2;
model.MC.depositionCriteria.maxMediumIdxToConsider = 2;
model.MC.depositionCriteria.evaluateOnlyAtEndOfLife = true;

Check by looking at photon paths that it actually works and only shows photons that dip beneath the depth of the medium transition.

To get the percentage of detected photons that got to that depth you'd need to run another simulation without the deposition criterion and take the ratio of the detected powers.

**Comment by ABoamfa** [comment]
Thank you for the elaborate explanation!
I have another question, is it possible to normalise the fluence rate not on the incident power, but on the detected power?
Thank you in advance!

**Reply by ankrh** [reply]
Both the NFR array and the detected power are normalized to the incident power, so you should be able to simply divide the NFR array by the detected power to get the normalization you want. Unless I misunderstood...

**Reply by ABoamfa** [reply]
Yes thank you, I figured it out :)
